js/script.js

```diff
@@ -0,0 +1,37 @@
+var quoteUrl = "https://quotesondesign.com/wp-json/posts?filter[orderby]=rand&filter[posts_per_page]=1";
+    tweetLink = "https://twitter.com/intent/tweet?text=",
+    prefix = "https://cors-anywhere.herokuapp.com/";
+
+function getQuote() {
+    $.getJSON(prefix + quoteUrl, createTweet);
+    $.ajaxSetup({ cache: false });
+}
+
+function createTweet(input) {
+
```

Comment:
before you continue your script, you should check that input is an array and contain at least one element inside